src/renderers/shared/fiber/ReactDebugCurrentFiber.js

```diff
@@ -25,10 +31,19 @@ function getCurrentFiberOwnerName() : string | null {
     if (fiber == null) {
       return null;
     }
-    if (fiber._debugOwner == null) {
-      return null;
+    switch (fiber.tag) {
+      case IndeterminateComponent:
+      case FunctionalComponent:
+      case ClassComponent:
+        return getComponentName(fiber);
+      case HostComponent:
+        if (fiber._debugOwner != null) {
+          return getComponentName(fiber._debugOwner);
+        }
+        return null;
+      default:
```

Comment:
Maybe coroutines but we haven't implemented them fully yet. I'm aiming for parity for now.